Tests/SotoCoreTests/Doc/RegionFromStringTests.swift

```diff
@@ -0,0 +1,28 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Soto for AWS open source project
+//
+// Copyright (c) 2017-2020 the Soto project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of Soto project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import SotoCore
+import XCTest
+
+class StringRegionTests: XCTestCase {
+    func testStringToRegion() {
```

Comment:
There is a `testRegionEnum` in `AWSClientTests` can you move that function here

Reply:
done. I am considering generating the RegionTest file automatically too, to improve test coverage